### PR TITLE
feat(blog): add Twitter share button to blog post

### PR DESCRIPTION
**Pull request by tadayosi**
Let's add Twitter share button to each blog post, so that we can share great articles more easily ;-)

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: layouts/blog/post.html [hunk 1]
```diff
@@ -20,6 +20,13 @@ <h1>{{ .Title }}</h1>
             </figure>
             {{ end }}
             {{ end }}
+            <a href="https://twitter.com/intent/tweet" class="twitter-share-button"
+                data-size="large"
+                data-text="{{ .Title }}"
+                data-url="{{ .Permalink }}"
+                data-hashtags="ApacheCamel"
+                data-show-count="false">Tweet this blog</a>
+            <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (zregvart) [inline layouts/blog/post.html:29]: Unfortunately this won't work with our content security policy. I'm also concerned about tracking and adding weight to the page (5 requests, ~400KB). Perhaps we could do https://sharingbuttons.io/ instead?

Comment (tadayosi) [reply layouts/blog/post.html:29]: Thanks Zoran. I was thinking there might be such a case. sharingbuttons.io looks great. So, according to our policy:
https://camel.apache.org/privacy-policy/
we should no embed a js script that tracks users' activities, and as sharingbuttons.io claims that it only uses HTML & CSS and does not track users' activities we can use the widget in the blog template, right?

**Comment by tadayosi** [comment]
OK, this time I used sharingbuttons.io to add three buttons (FB/Twitter/LinkedIn, my opinionated choice). I also added "Share this blog" label before the buttons as otherwise I noticed it may wrongly signal they look SNS links for the author.

File: layouts/blog/post.html [hunk 2]
```diff
@@ -20,6 +20,33 @@ <h1>{{ .Title }}</h1>
             </figure>
             {{ end }}
             {{ end }}
+
+            <h4>Share this blog</h4>
+
+            <!-- Sharingbutton Facebook -->
+            <a class="resp-sharing-button__link" href="https://facebook.com/sharer/sharer.php?u={{ .Permalink }}" target="_blank" rel="noopener" aria-label="">
```
Comment (zregvart) [inline layouts/blog/post.html:27]: I need to test this, but I think we could use the `aria-label` here to help folk with sight impairment. Same goes for other buttons.

```suggestion
            <a class="resp-sharing-button__link" href="https://facebook.com/sharer/sharer.php?u={{ .Permalink }}" target="_blank" rel="noopener" aria-label="Share on Facebook">
```

File: antora-ui-camel/src/css/sharingbuttons.css [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+.resp-sharing-button--twitter {
+  background-color: #55acee;
+  border-color: #55acee;
+}
+
+.resp-sharing-button--twitter:hover {
+  background-color: #2795e9;
+}
+
+.resp-sharing-button--pinterest {
```
Comment (zregvart) [inline antora-ui-camel/src/css/sharingbuttons.css:55]: There are bits that we don't use here (pinterest being one of them), it's not a real issue, couple of bytes more in the CSS, but perhaps we can remove those as we're not using them?

**Review by zregvart — COMMENTED** [review]
Looks good, thanks, let's wait for the checks and preview.

File: antora-ui-camel/src/css/sharingbuttons.css (hunk 3 above)
Comment (tadayosi) [reply antora-ui-camel/src/css/sharingbuttons.css:55]: I kept them as-is thinking others might want to add Pinterest share button (or etc.) later :smiley: but I'm fine with trimming unused ones.

File: layouts/blog/post.html (hunk 2 above)
Comment (tadayosi) [reply layouts/blog/post.html:27]: Agreed. I'll add them.

**Comment by tadayosi** [comment]
Looking good, isn't it?  Now I'll apply Zoran's feedback and update it before merge.
Screenshot from 2021-05-19 17-26-51

**Pull request merged**